upgrading/automated_upgrades.adoc

```diff
@@ -546,12 +546,26 @@ OpenShift upgrades] for further guidance.
 [[special-considerations-for-mixed-environments]]
 ==== Special considerations for mixed environments
 
-Before you upgrade a mixed environment, for example one with Red Hat Enterprise
+Before you upgrade a mixed environment, such as one with Red Hat Enterprise
 Linux (RHEL) and RHEL Atomic Host, set values in the inventory file
 for both the `openshift_pkg_version` and `openshift_image_tag` parameters.
-Setting these variables ensures that all nodes in your cluster run the same
+Setting these values ensures that all nodes in your cluster run the same
 version of {product-title}.
 
+For example, to upgrade from {product-title}
+3.9 to {product-title} 3.10, you would set the values as follows:
```

Comment:
s/you would set the values as follows/set the following parameters and values:

>!  would v
Use only to refer to an unfulfilled condition or hypothesis and to express the future in the past... Do
not use as an auxiliary verb...

Reply:
Done. Thanks.